src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -19,7 +19,7 @@ redirects:
 
 We monitor and collect [metrics](http://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#metrics) from the following control plane components:
 
-* **ETCD**: leader information, resident memory size, number of OS threads, consensus proposals data, etc. For a list of supported metrics, see [ETCD data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#etcd-data).
+* **etcd**: leader information, resident memory size, number of OS threads, consensus proposals data, etc. For a list of supported metrics, see [ETCD data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#etcd-data).
```

Comment:
There are mote `ETCD`. `sed` it! (after `grep`ing to be sure)

Reply:
https://github.com/newrelic/docs-website/pull/5742/commits/1f0e2293e24b452f8a8b316161134de8f0526623 :done: